src/components/cursor-controller.js

```diff
@@ -132,18 +134,38 @@ AFRAME.registerComponent("cursor-controller", {
       camera.object3D.getWorldPosition(cameraPos);
       cameraPos.y = cursor.object3D.position.y;
       cursor.object3D.lookAt(cameraPos);
+      cursor.object3D.scale.setScalar(Math.pow(this.distance, 0.315) * 0.75);
```

Comment:
How very magic. I am guessing you just eyeballed something that looked ok?

Reply:
yeeeep